src/web/make-report.rkt

```diff
@@ -61,9 +61,12 @@
     (for/sum ([t tests]) (or (table-row-start t) 0)))
   (define total-result
     (for/sum ([t tests]) (or (table-row-result t) 0)))
-  (define total-bits
-    (for/sum ([t tests])
-      (representation-total-bits (get-representation (table-row-precision t)))))
+  (match-define
+    (list
+     (list _ maximum-accuracy)
+     (list _ initial-accuracy)
+     frontier)
+    (merged-cost-accuracy tests))
```

Comment:
I'm worried about this: `merged-cost-accuracy` can take a long time to run. Don't we already have the results somewhere in the `report-info`?

Reply:
I think putting it in `report-info` sounds reasonable.